go-tour/stringer.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+// IPAddr type represents ip-adress
+type IPAddr [4]byte
+
+func (ip IPAddr) String() string {
+	str := fmt.Sprintf("%v", ip[0])
+	for _, b := range ip[1:] {
+		str += fmt.Sprintf(".%v", b)
+	}
+	return str
+}
+
+// TODO: Add a "String() string" method to IPAddr.
+func main() {
+	hosts := map[string]IPAddr{
+		"loopback":  {127, 0, 0, 1},
+		"googleDNS": {8, 8, 8, 8},
+	}
+	for name, ip := range hosts {
+		fmt.Printf("%v: %v\n", name, ip)
```

Comment:
make a benchmark for comparing your implementation with another one with strconv package, please